game/setup.lua

```diff
@@ -41,7 +42,9 @@ function setup.config()
     CAM = Camera(O_WIN_W/2, O_WIN_H/2) --Set camera position to center of screen
 
     --IMAGES--
+    local path = "assets/imgs/"
     IMG = { --Table containing all the images
+      CARD_BACK_1 = love.graphics.newImage(path.."cards/card_back_1.png")
```

Comment:
i'll let this slide on account that i also made a hardcoded image asset loading on the `sectorview`, but we should use the DB for loading textures from image assets